cli/how-to-deploy-rest.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+# <installation>
+# az extension add -n ml
+# </installation>
+
+# <create environment variables>
+SUBSCRIPTION_ID="7ab7d5bc-5d9e-47ef-80e6-2dffa8ca83a1"
+RESOURCE_GROUP="trmccorm-centraluseuap"
+WORKSPACE="trmccorm-centraluseuap"
+API_VERSION="2021-03-01-preview"
+
+TOKEN=$(az account get-access-token | jq -r ".accessToken")
+
+AZURE_STORAGE_ACCOUNT="trmccormcentra1277620275"
```

Comment:
Is there a way to retrieve these 3 variables (associated with workspace) instead of having user specify them?

Reply:
Not that I know of